frontend/app/src/pages/main/tenant-settings/api-tokens/components/create-token-dialog.tsx

```diff
@@ -5,16 +5,30 @@ import {
 } from '@/components/ui/dialog';
 import { Button } from '@/components/ui/button';
 import { z } from 'zod';
-import { useForm } from 'react-hook-form';
+import { Controller, useForm } from 'react-hook-form';
 import { zodResolver } from '@hookform/resolvers/zod';
 import { Label } from '@/components/ui/label';
 import { Input } from '@/components/ui/input';
 import { cn } from '@/lib/utils';
 import { Spinner } from '@/components/ui/loading';
 import { CodeHighlighter } from '@/components/ui/code-highlighter';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+
+export const EXPIRES_IN_OPTS = {
```

Comment:
potentially we could add `const DAY = 24 * 60 * 60` and then 3 * DAY for simplicity, but it's not a big deal

Reply:
should fix the underlying validator... will leave for now